src/components/common/DynamicTabs/DynamicTabs.scss

```
@@ -149,3 +149,19 @@
         }
     }
 }
+
+.dynamic-tabs-select-popup-body {
+    .dynamic-tabs-select__menu {
+        margin-top: 0px;
+
+        border-radius: 0 0 4px 4px;
+    }
+
+    .dynamic-tabs-select__control {
+        border-radius: 4px 4px 0 0;
+    }
+
+    .dynamic-tabs-select__indicator {
+        display: none;
+    }
```

Check for indentation
```suggestion
    .dynamic-tabs-select { &__menu {
        margin-top: 0px;
        }

        border-radius: 0 0 4px 4px;
    }

    .dynamic-tabs-select__control {
        border-radius: 4px 4px 0 0;
    }

    .dynamic-tabs-select__indicator {
        display: none;
    }
```